tests/elastic_stabilization_time_scale_factor/original.prm

```diff
@@ -0,0 +1,40 @@
+# This test checks whether the viscoelastic finite-length elastic plate
```

Comment:
please remove this file and all other parameters files. The idea is to only have files in the directory that show some model result that lets us determine if the test output has changed (typical examples are `screen-output` and `statistics`)